sdk/videoanalyzer/tests.yml

```diff
@@ -0,0 +1,6 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: mediaservices
```

Comment:
"videoanalyzer". Please be sure to clean up any track 1 info you copied from the older directory and remove any non-GA'd libraries like I mentioned above to avoid confusion.